README.md

```diff
@@ -82,7 +82,9 @@ A high level breakdown of the components of the framework:
  * The [Tools/Pipeline](Tools/Pipeline) tool is a GUI frontend for content processing.
 
 
-You can generate solutions for platforms that are not buildable from the current OS with: `mono Protobuild.exe --generate $PLATFORM`.
+You can generate solutions for platforms that are not buildable from the current OS with: 
+ - Windows: `.\Protobuild.exe --generate $PLATFORM`
+ - Linux or Mac: `mono Protobuild.exe --generate $PLATFORM`
```

Comment:
Move this up into the first list in this section right after the first Protobuild mention.